pkg/oci/remote/write.go

```diff
@@ -127,3 +133,74 @@ func WriteAttestations(repo name.Repository, se oci.SignedEntity, opts ...Option
 	// Write the Signatures image to the tag, with the provided remote.Options
 	return remoteWrite(tag, atts, o.ROpt...)
 }
+
+// WriteSignaturesExperimentalOCI publishes the signatures attached to the given entity
+// into the provided repository (using OCI 1.1 methods).
+func WriteSignaturesExperimentalOCI(d name.Digest, se oci.SignedEntity, opts ...Option) error {
```

Comment:
this method also suffers the same problem as Referrers, needing to use makeOptions